meal-planner/src/components/MealSelect.js

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components'
+
+export default function MealSelect({ key, onClick }) {
+    const meals = [
+        'Käsestulle',
+        'Kartoffelsalat',
+        'Gemüselasagne',
+        'Mayo',
+        'Currywurst',
+        'Linsensuppe',
+        'Frikassee',
+        'Currywurst',
+        'Katzenfutter',
+    ]
+
+    function onMealClick(meal) {
+        console.log(meal)
+    }
+
+    return (
+        <ListStyled>
+            {meals.map((meal, index) => (
+                <li key={index} onClick={() => onMealClick(meal)}>
+                    {meal}
+                </li>
+            ))}
+        </ListStyled>
+    )
+}
+
+const ListStyled = styled.ul`
+    display: flex;
+    height: 100px;
+    list-style: none;
+    overflow: auto;
+
+    li {
+        display: grid;
+        background: maroon;
+        margin: 5px;
+        color: ivory;
+        justify-content: center;
```

Comment:
Here as well, some ordering would it make easier to grasp. E. g. all properties regarding display: grid could go together.